<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountsPageReceivingTab</class>
 <widget class="QWidget" name="AccountsPageReceivingTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>322</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="vlay">
     <item>
      <widget class="PimCommon::ManageAccountWidget" name="mAccountsReceiving">
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="customizeAccountOrder">
       <property name="text">
        <string>Customize accounts order...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group">
       <property name="title">
        <string>New Mail Notification</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="mBeepNewMailCheck">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Beep</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mVerboseNotificationCheck">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Show for each folder the number of newly arrived messages</string>
          </property>
          <property name="text">
           <string>Deta&amp;iled new mail notification</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mOtherNewMailActionsButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Other Actio&amp;ns...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>PimCommon::ManageAccountWidget</class>
   <extends>QWidget</extends>
   <header>pimcommon/widgets/manageaccountwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
